api/src/main/java/io/opencensus/trace/BlankSpan.java

```diff
@@ -34,6 +34,10 @@ private BlankSpan() {
     super(SpanContext.INVALID, null);
   }
 
+  /** No-op implementation of the {@link Span#addAttribute(String, AttributeValue)} method. */
+  @Override
+  public void addAttribute(String key, AttributeValue value) {}
```

Comment:
should this be putAttribute? what happens if you call this twice with the same key

Reply:
Put is good only for Java developer, C# (they have an add method, but fails if key already there), C++ has insert and emplace, Python has "set" (not as a literal function, but they use it in explanation), Go does seem to only support direct access "m["route"] = 66" but they also use "set" to explain the action.

Seems to me that set is the winner form all these languages.

Reply:
I think this should be language specific. putAttribute is the right name for Java and set for python, etc.